tests/services/analysis_starter/test_subprocess_submitter.py

```diff
@@ -1,36 +1,56 @@
 import subprocess
-from unittest import mock
 
+import pytest
+from pytest_mock import MockerFixture
+
+from cg.constants.priority import SlurmQos
 from cg.services.analysis_starter.configurator.models.microsalt import MicrosaltCaseConfig
-from cg.services.analysis_starter.submitters.subprocess.submitter import SubprocessSubmitter
+from cg.services.analysis_starter.configurator.models.mip_dna import MIPDNACaseConfig
+from cg.services.analysis_starter.submitters.subprocess.submitter import (
+    SubprocessCaseConfig,
+    SubprocessSubmitter,
+)
 
 
-def test_subprocess_submitter():
+@pytest.mark.parametrize(
+    "case_config",
+    [
+        MicrosaltCaseConfig(
+            case_id="case_id",
+            binary="binary",
+            conda_binary="conda_binary",
+            config_file="config_file",
+            environment="environment",
+            fastq_directory="fastq_directory",
+        ),
+        MIPDNACaseConfig(
+            case_id="case_id",
+            pipeline_binary="pipeline_binary",
+            conda_binary="conda_binary",
+            pipeline_config_path="pipeline_config_path",
+            conda_environment="conda_environment",
+            slurm_qos=SlurmQos.NORMAL,
+            use_bwa_mem=False,
+            email="my@email.se",
+        ),
+    ],
+    ids=["microSALT", "MIP-DNA"],
+)
+def test_subprocess_submitter(case_config: SubprocessCaseConfig, mocker: MockerFixture):
```

Comment:
I guess the name of the test should have the name of the tested function?
```suggestion
def test_subprocess_submitter_submit(case_config: SubprocessCaseConfig, mocker: MockerFixture):
```